plugin-server/src/cdp/utils.ts

```diff
@@ -50,10 +50,6 @@ export function convertToHogFunctionInvocationGlobals(
     siteUrl: string
 ): HogFunctionInvocationGlobals {
     const properties = event.properties ? JSON.parse(event.properties) : {}
-    if (event.elements_chain) {
-        properties['$elements_chain'] = event.elements_chain
```

Comment:
This is the only change im a bit wary of. Its cobwebs of memories but i know there was some worry that existing destinations had this included so if we are pulling it out it _could_ break something downstream. Hard to judge how likely that is though.

Reply:
There were some existing destinations, like the google cloud storage plugin, which created a "one-line-csv" with separate columns for different fields. Sadly that is going to be a manual migration anyway, since we're also not sending `properties.$set` and `properties.$set_once` as top level fields in the new function anyway.

I'll go through each destination and make sure if they're exporting the event, they will now include this new field.